src/tests/radmin/all.mk

```diff
@@ -0,0 +1,148 @@
+#
+#	Unit tests for radmin
+#
+
+#
+#	Test name
+#
+TEST := test.radmin
+
+#
+#	Config settings
+#
+PORT               := 12341
+BIN_PATH           := $(BUILD_DIR)/bin/local
```

Comment:
A lot of this is already defined in Make.inc

TESTBINDIR - directory where the binary files are located
TESTBIN - path (including jlibtool arguments) for running program